code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -99,7 +99,7 @@
 
 /obj/item/reagent_containers/hypospray/medipen
 	name = "epinephrine medipen"
-	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body."
+	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body, and stop the production of histamine during an allergic reaction."
```

Comment:
I'm uh, kinda behind the times, but do we A: have allergies, and B: does the medipen actually fix them/how

Reply:
Extreme medicine allergy is a quirk we have, and it stop the production of histamine for a time so you can go see a doctor ans not die.

Reply:
is it the preservative that stops the production of histamine? I guess there's not a great way to fix that

Reply:
The formaldehyde stops organ decay, the ephedrine stops histamine production and stabilizes people in crit